src/cmd2.cc

```diff
@@ -717,41 +657,6 @@ static void chest_death(int y, int x, s16b o_idx)
 
 
 /*
- * Chests have traps too.
```

Comment:
I'm just sticking this comment here, because I've experienced this in my hacked-up "don't generate traps" branch: Do you also experience chests always being marked as "(empty)" when playing with these changes?